File: 프로그래머스/0/181836. 그림 확대/그림 확대.java
class Solution {
    public String[] solution(String[] picture, int k) {
        String[] answer = new String[picture.length * k];
        int index = 0;
        for(int i = 0; i < picture.length; i++) {
            String s = "";
            for(int j = 0; j < picture[i].length(); j++) {          
                for(int n = 0; n < k; n++) {            
                    s += picture[i].charAt(j); 
                }
            }
            for(int c = 0; c < k; c++) {
                answer[index++] = s;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181836. 그림 확대/그림 확대.js
function solution(picture, k) {
    const answer = [];
    const tempArr = picture.map(x => x.split('').map(x1 => x1.repeat(k)).join(''));
    
    for (let i=0; i<tempArr.length; i++) {
        for(let j=0; j<k; j++){
            answer.push(tempArr[i]);
        }
    }
    return answer;
}